### PR TITLE
Don't download attachments in paralell and in chunks anymore

### PR DESCRIPTION
**Pull request by tomas-stefano**
## Context

The download in chunks and in paralell is raising many errors
in production (0 status code from filestore).

Changing the implementation to a simple one to understand better
what is going on.

## Result

All files are appearing in the email (results from Submitter downloading from Filestore)

Screenshot 2021-09-22 at 08 52 58

## Logs

You can see an example on the [Kibana logs here](https://kibana.cloud-platform.service.justice.gov.uk/_plugin/kibana/app/kibana#/discover?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:now-15m,to:now))&_a=(columns:!(_source),filters:!(),index:d4959120-0186-11ec-8311-8b9e5a9c1db5,interval:auto,query:(language:lucene,query:'%22Filestore%20Download%20API%20request%22'),sort:!(!('@timestamp',desc))))

### CODE REVIEW

File: app/services/download_attachments.rb [hunk 1]
```diff
@@ -0,0 +1,63 @@
+class DownloadAttachments
+  SUBSCRIPTION = 'download_attachments'.freeze
+  TIMEOUT = 30
+  OPEN_TIMEOUT = 30
+  attr_reader :attachments, :target_dir, :token, :access_token
+
+  def initialize(attachments:, target_dir: nil, token:, access_token:, jwt_skew_override:)
+    @attachments = attachments
+    @target_dir = target_dir
```
Comment (brenetic) [inline app/services/download_attachments.rb:9]:
```suggestion
    @target_dir = target_dir || Dir.mktmpdir
```

just a thought

File: app/services/download_attachments.rb [hunk 2]
```diff
@@ -0,0 +1,63 @@
+class DownloadAttachments
+  SUBSCRIPTION = 'download_attachments'.freeze
+  TIMEOUT = 30
+  OPEN_TIMEOUT = 30
+  attr_reader :attachments, :target_dir, :token, :access_token
```
Comment (brenetic) [inline app/services/download_attachments.rb:5]: would it be worth us taking the opportunity to rename `token` here to better describe what it is?

**Review by brenetic — APPROVED** [review]
🚀 

great stuff dude! hoping these log outputs will help us to diagnose issues going forward

is 30 seconds timeout ok? it's all on a queue i guess . . .

File: app/services/download_attachments.rb (hunk 2 above)
Comment (tomas-stefano) [reply app/services/download_attachments.rb]: Done!

File: app/services/download_attachments.rb (hunk 1 above)
Comment (tomas-stefano) [reply app/services/download_attachments.rb]: Done

**Comment by tomas-stefano** [comment]
@brenetic I think it is okay. Every download is taking roughly 1 second (1 MB images). So 30 is a good one.
Just to clarify this solution won't solve a problem that it is not app specific.
But if submitter can't connect for 30 seconds the open timeout will raise and the code will raise `Faraday::ConnectionFailed`
If submitter connect to filestore and for some reason Filestore is taking too long to process the request, the code will raise `Faraday::TimeoutError`. 
With this we can see the underlying issue *if happens again*.

**Review by brenetic — APPROVED** [review]
🚀

**Pull request merged**